packages/@ama-sdk/core/jest.config.js

```diff
@@ -1,12 +1,10 @@
-const getJestConfig = require('../../../jest.config.ut').getJestConfig;
+const getJestGlobalConfig = require('../../../jest.config.ut').getJestGlobalConfig;
 
 /** @type {import('ts-jest/dist/types').JestConfigWithTsJest} */
 module.exports = {
-  ...getJestConfig(__dirname, false),
-  displayName: require('./package.json').name,
-  fakeTimers: {
-    enableGlobally: true,
-    // TODO try to make date utils work with fake Date
-    doNotFake: ['Date']
-  }
+  ...getJestGlobalConfig(),
+  projects: [
```

Comment:
Does the VsCode Jest plugin still working?
I am a bit afraid because it is using the root `jest.config.js` which is already defining a project list that is now pointing to other jest's project config

Reply:
I had to change the pattern on the root jest config to `'testing/jest.config.*.{j,t}s'` but it now works on VSCode